sim_pipeline/halos/get_halos.ipynb

```diff
@@ -0,0 +1,166 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    print('kappa at center is: ',kappa)
</pre><p>Great notebook! The kappa value seems very high. Did you subtract the mean kappa of random drawn distributions?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sibirrer/sim-pipeline/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Currently, I haven't performed the random draws, but I'll make sure to include them in my upcoming work.

Reply:
> Line #8.    print('kappa at center is: ',kappa)
> Great notebook! The kappa value seems very high. Did you subtract the mean kappa of random drawn distributions?
> 
> 
> _Reply via [ReviewNB](https://app.reviewnb.com/sibirrer/sim-pipeline/pull/7/discussion/#comment-1279307843)_

Currently, I haven't performed the random draws, but I'll make sure to include them in my upcoming work.